featuretools/synthesis/get_valid_primitives.py

```diff
@@ -0,0 +1,93 @@
+from featuretools.primitives import AggregationPrimitive, TransformPrimitive
+from featuretools.primitives.utils import (
+    get_aggregation_primitives,
+    get_transform_primitives
+)
+from featuretools.synthesis.deep_feature_synthesis import DeepFeatureSynthesis
+from featuretools.synthesis.utils import (
+    _categorize_features,
+    get_unused_primitives
+)
+from featuretools.utils.gen_utils import Library
+
+
+def get_valid_primitives(entityset, target_entity, max_depth=2, selected_primitives=None):
```

Comment:
This can probably be a separate PR, but we need to update this function to remove references to  `entity`. Can you create an issue to capture this if you don't update it as part of this PR?

Reply:
Sounds good. I'll create issues for tracking

Reply:
Filed #1494